net/libntlm/Portfile

```diff
@@ -1,24 +1,37 @@
-PortSystem 1.0
-
-name		libntlm
-version		1.2
-revision	0
-categories	net security devel
-license		LGPL-2.1+
-maintainers	nomaintainer
-description	Implement Microsoft's NTLM authentication
-long_description	\
-	NTLM employs a challenge-response mechanism for 	\
-	authentication, in which clients are able to prove 	\
-	their identities without sending a password to the 	\
-	server. It consists of three messages, commonly 	\
-	referred to as Type 1 (negotiation), Type 2 		\
-	(challenge) and Type 3 (authentication).
-homepage	http://josefsson.org/libntlm
-master_sites	http://josefsson.org/libntlm/releases
-platforms	darwin
-checksums	md5 13b40af721b29005652fb429a3ae9582 \
-		sha1 27538a3375690a37574fa991fbd327d150d8b505 \
-		rmd160 b7344213f2510c93e30583fc567961a4e6aedd13
-configure.args	--infodir=${prefix}/share/info \
-		--mandir=${prefix}/share/man
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libntlm
+version             1.6
+revision            0
+categories          net security devel
+license             LGPL-2.1+
+maintainers         nomaintainer
+
+description         Implement Microsoft's NTLM authentication
+long_description    NTLM employs a challenge-response mechanism for     \
+                    authentication, in which clients are able to prove  \
+                    their identities without sending a password to the  \
+                    server. It consists of three messages, commonly     \
+                    referred to as Type 1 (negotiation), Type 2         \
+                    (challenge) and Type 3 (authentication).
+
+homepage            https://www.nongnu.org/libntlm
+master_sites        ${homepage}/releases
+
+platforms           darwin
```

Comment:
`platforms darwin` is now implicit, and no longer needs to be specified when specifically for `darwin` only.

Reply:
Thanks @herbygillot. This is fixed.